starfinder/db/models.py

```diff
@@ -313,7 +433,7 @@ class RacialTrait(db_engine.Model, ModelBase, HasId):
     name = db_engine.Column(db_engine.String(64), nullable=False)
     description = db_engine.Column(db_engine.String(64), nullable=False)
 
-    trait_modifiers = orm.relationship('TraitModifier', backref='racial_trait')
+    racial_trait_modifiers = orm.relationship('RacialTraitModifier', backref='racial_trait')
```

Comment:
good catch! This looked wrong to me at first, but I see that you changed the name elsewhere & came up here to keep everything aligned. nice job.